torch/utils/data/datapipes/_typing.py

```diff
@@ -474,7 +474,7 @@ def reinforce_type(self, expected_type):
     hint to restrict the type requirement of DataPipe instance.
     """
     if isinstance(expected_type, tuple):
-        expected_type = Tuple[expected_type]
+        expected_type = tuple[expected_type]
```

Comment:
Since we're planning to deprecate and then delete datapipes here, I'm OK with breaking lint, but not 100% sure of the downstream implications